### PR TITLE
Signup page merge ready

### PR DESCRIPTION
**Pull request by b0llu**
Only check signup.html and signup.css

image

### CODE REVIEW

**Comment by Dev-AkshatGupta** [comment]
This seems great to be merged buddy :)

File: signup-page/signup.css [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    gap: 10px;
+    background-color: #d8dcdf;
+    color: var(--login-color);
+    border-radius: 5px;
+}
+
+.login-container h1 {
+    font-size: 2rem;
+    color: var(--text-black);
+}
```
Comment (manojsarna) [inline signup-page/signup.css:28]: looks good!!

File: signup-page/signup.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    height: 100%;
+    font-size: larger;
+}
+
+.login-container {
+    display: flex;
+    flex-direction: column;
+    padding: 3rem 4rem;
+    gap: 10px;
+    background-color: #d8dcdf;
```
Comment (manojsarna) [inline signup-page/signup.css:20]: use rems and px wisely

File: signup-page/signup.css [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    display: block;
+    border-radius: 5px;
+    padding: 5px 2px;
+    border: 1px solid var(--content-color);
+}
+
+::placeholder {
+    padding: 0 12px;
+    font-family: "Outfit";
+    letter-spacing: 1px;
```
Comment (manojsarna) [inline signup-page/signup.css]: check to use rems or px

File: signup-page/signup.css [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+}
+
+@media screen and (max-width: 850px) {
+    html {
+        font-size: 50%;
+    }
+
+    .landing-container {
+        grid-template-columns: repeat(1, 25rem) repeat(2, 1fr);
+        grid-template-areas:
```
Comment (manojsarna) [inline signup-page/signup.css:88]: good media queries!!

File: signup-page/signup.css [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+    .navbar .btn {
+        margin-right: 1rem;
+    }
+
+    .name {
+        margin: 0;
+    }
+
+    .margin-left-auto {
```
Comment (manojsarna) [inline signup-page/signup.css:134]: good work!!

File: wishlist-page/wishlist.css [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.card-container.product-card {
+    width: 30rem;
+}
+
+.card-container .fas {
+    color: var(--alert-color);
+    cursor: pointer;
+    border-radius: 50%;
```
Comment (manojsarna) [inline wishlist-page/wishlist.css]: looks great!!

**Review by manojsarna — APPROVED** [review]
Overall looks good!!

You can merge it!!

File: signup-page/signup.css (hunk 3 above)
Comment (b0llu) [reply signup-page/signup.css]: fixed

File: cart-page/cart.css [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+    border: none;
+    height: 5rem;
+}
+
+.ammount-container .card-container .btn:hover {
+    background-color: var(--order-btn);
+    transition: 0.2s ease-in-out;
+}
+
+.ammount-container h2 {
```
Comment (vashnavichauhan1825) [inline cart-page/cart.css]: spelling correction !

**Review by vashnavichauhan1825 — CHANGES_REQUESTED** [review]

File: cart-page/cart.css (hunk 7 above)
Comment (b0llu) [reply cart-page/cart.css]: fixed it thanks for pointing it out

**Pull request merged**